interpreter/cling/lib/Interpreter/DefinitionShadower.cpp

```diff
@@ -46,6 +46,15 @@ namespace cling {
     return true;
   }
 
+  // \brief Returns whether the given declaration is a template instantiation.
+  static bool isTemplateInstantiation(const Decl *D) {
```

Comment:
Should we include template variable instantiations?

Reply:
Yes.  Thanks, @vgvassilev.  :-)